src/containers/hashmap.d

```diff
@@ -85,12 +85,26 @@ struct HashMap(K, V, Allocator = Mallocator, alias hashFunction = generateHash!K
 		}
 	}
 
-	~this()
+	~this() nothrow
 	{
-		import stdx.allocator : dispose;
+		try
+			clear();
+		catch (Throwable)
+			assert(false);
```

Comment:
`scope(failure) assert (0);` is the idiomatic way though that's nit picking

Reply:
Makes sense, but this is the first time I've seen that suggested.